src/fancyrag/splitters/caching_fixed_size.py

```
@@ -172,13 +172,24 @@ async def run(
         Returns:
             TextChunks: The resulting chunks for `text`. On cache hits, chunks preserve text/index/metadata but have newly generated `uid` values.
         """
+        is_sequence_input = not isinstance(text, str)
+        normalized_sequence: tuple[str, ...] | None = None
+        sequence_items: list[str] | None = None
+        if is_sequence_input:
+            normalized_sequence = tuple(text)  # type: ignore[arg-type]
+            sequence_items = list(normalized_sequence)
```

Ruff (0.13.3)

Local variable `sequence_items` is assigned to but never used

Remove assignment to unused variable `sequence_items`

(F841)